pandas/core/internals/managers.py

```diff
@@ -1474,21 +1387,123 @@ def take(self: T, indexer, axis: int = 1, verify: bool = True) -> T:
             consolidate=False,
         )
 
-    def _equal_values(self: T, other: T) -> bool:
+
+class BlockManager(_BlockManager):
+    """
+    _BlockManager that holds 2D blocks.
+    """
+
+    ndim = 2
+
+    def __init__(
+        self,
+        blocks: Sequence[Block],
+        axes: Sequence[Index],
+        verify_integrity: bool = True,
+    ):
+        self.axes = [ensure_index(ax) for ax in axes]
+        self.blocks: tuple[Block, ...] = tuple(blocks)
+
+        for block in blocks:
+            if self.ndim != block.ndim:
+                raise AssertionError(
+                    f"Number of Block dimensions ({block.ndim}) must equal "
+                    f"number of axes ({self.ndim})"
+                )
+
+        if verify_integrity:
+            self._verify_integrity()
+
+        # Populate known_consolidate, blknos, and blklocs lazily
+        self._known_consolidated = False
+        # error: Incompatible types in assignment (expression has type "None",
+        # variable has type "ndarray")
```

Comment:
why do you need all these?

Reply:
the attributes or the "type ignore"s?

Reply:
the type ignores, but certainly can be in a followup